tests/Functional/DiffTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Tests\Functional;
+
+use Manala\Command\Diff;
+use Manala\Env\EnvEnum;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class DiffTest extends \PHPUnit_Framework_TestCase
+{
+    const ACME_PROJECT_WD = MANALA_DIR.'/tests/fixtures/Command/DiffTest/AcmeProject';
+    const EXPECTED_OUTPUT_FILE = MANALA_DIR.'/tests/fixtures/Command/DiffTest/expected_acme.patch';
```

Comment:
> As soon as we always install the exact same version.

Not sure to understand here, we manalize the temporary project with the same codebase as the one used to check the diff, so the pack is always up to date on both sides, right?

Also I find worst to have an unexisting env in our code base, but let's keep it (we have to quickly add the ability to load remote packs in order to remove it ^^)